opm/simulators/flow/BlackoilModelEbos.hpp

```diff
@@ -39,7 +39,6 @@
 
 #include <opm/grid/UnstructuredGrid.h>
 #include <opm/simulators/timestepping/SimulatorReport.hpp>
-#include <opm/simulators/linalg/ParallelIstlInformation.hpp>
```

Comment:
Why? Are there other headers we do not install?
Maybe someone external wants to use the functionality. How would she proceed?

Reply:
Why the include is removed? Because it is not used.

Why the separate compile unit?The goal is to pull less istl into compile units. If you think there may be external users, I can use an impl file in addition to the cpp file.

Reply:
:flushed: Somehow thought this was a change to CMake_files.txt. No idea why was under that impression. Sorry for the noise.